### PR TITLE
Add com.vixalien.decibels

### PR DESCRIPTION
**Pull request by vixalien**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

**Comment by hfiguiere** [comment]
bot, build com.vixalien.decibels

**Comment by vixalien** [comment]
the build seems to have stalled. maybe you can try re-running it again?

**Comment by vixalien** [comment]
bot, build com.vixalien.decibels

**Comment by vixalien** [comment]
bot, build com.vixalien.decibels

**Comment by vixalien** [comment]
@hfiguiere ptal

**Comment by vixalien** [comment]
bot, build com.vixalien.decibels

File: com.vixalien.decibels.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
+{
+  "id": "com.vixalien.decibels",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "45",
+  "sdk": "org.gnome.Sdk",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.node18",
+    "org.freedesktop.Sdk.Extension.typescript"
```
Comment (vixalien) [inline com.vixalien.decibels.json:8]:
```suggestion
    "org.freedesktop.Sdk.Extension.typescript"],
```

**Comment by vixalien** [comment]
bot, build com.vixalien.decibels

**Comment by vixalien** [comment]
@barthalion ptal

**Comment by vixalien** [comment]
@hfiguiere I think it's ready. can you take a look?

**Comment by vixalien** [comment]
it works for me.

I can't reinstall the app to test, I think the link has expired. let me try a rebuild

**Comment by vixalien** [comment]
bot, build com.vixalien.decibels

**Comment by vixalien** [comment]
welp, I can't install the app. I'm not sure why

**Comment by bbhtt** [comment]
> The command in the desktop file doesn't work:
> 
> ```
> ~/.l/s/f/e/s/applications ❱ /usr/bin/flatpak run --branch=test --arch=x86_64 --command=com.vixalien.decibels com.vixalien.decibels
> error: Could not connect: No such file or directory
> ```

Works fine on my end too. https://github.com/flathub/flathub/pull/4529#issuecomment-1740611629

**Pull request closed without merging**